app/scripts/metamask-controller.js

```diff
@@ -840,6 +840,11 @@ export default class MetamaskController extends EventEmitter {
       ],
     });
 
+    const allowLocalSnaps = process.env.ALLOW_LOCAL_SNAPS;
+    const requireAllowlist = process.env.REQUIRE_SNAPS_ALLOWLIST;
+    assert(typeof allowLocalSnaps === 'boolean');
+    assert(typeof requireAllowlist === 'boolean');
```

Comment:
Could we possibly move these assertions to higher up in the execution layer? It feels weird to be doing assertions on build variables in the application logic.

Reply:
Yeah, I removed these assertions for now. They also broke a bunch of tests